students/11th/changsik/westagram/user/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import Signup
+
+urlpatterns = [
+    path('', Signup.as_view())
```

Comment:
회원가입 엔드포인트 이므로 엔드포인트주소에 sign-up 을 써주는편이 호출하는 편에서도 명확하고 좋을 것 같습니다.
그리고 Signup은 클래스는 보통 CamelCase로 쓰기 때문에 아래와 같이 수정하는게 좋을 것 같습니다. View를 붙인이유는 모델과 구분을 명확하게 하는이유도있습니다.
```suggestion
    path('', SignUpView.as_view())
```